### ISSUE TITLE
[0.9.0]: Losing measurements

### ISSUE THREAD

**Issue by BigEpsilon** [influxdb/influxdb#3118]
Hi,

I have a serious problem with the current version of influxdb on ubuntu (14.04 LTS, one machine) with both versions installed from source (master branch) and the deb package. I have several databases, and each day or 2 days, randomly, I lose all the data of one or many databases. The database is still here, but with nothing in it, all measurement and series disapears. This hapens regularly, with many databases, and even multiples time on the same database.
Nothing appears in the logs, the data just disapear.
And additionnaly, because there is a bug that makes influxdb crashs when you do a select on a non existing measurement, the database crashs after loosing the data if you try to access them.
I dont know haw what more useful informations I can give, but this bug is quite misterious.

**Comment by beckettsean** [comment]
@BigEpsilon can you provide your logs and your configuration file in a gist? Are there any steps you can suggest for us to try and reproduce this? It's a serious issue but there's not much here to go on in terms of recreating it.

It sounds like you've switched back and forth between a build from master and a versioned package. Is that true? Did you switch from 0.9.0 to master, or from master to 0.9.0, or both?

**Comment by BigEpsilon** [comment]
Hi,
Unfortunaly I have no idea how I can reproduce this. Here are my configuration file and logs:
<script src="https://gist.github.com/BigEpsilon/81dd661cf62f4be1fe64.js"></script>
In the logs, all the python calls are me recreating the databases after I lost them. All the nil pointer errors are a select to a measurement that have been lost.

On that machine I switched between many version, strating from 0.8, then the 0.9 rcx (deb install), the the official 0.9 deb package. When I started experimenting this bug, I installed go and compiled from source in order to see if the bug was corrected.
